charts/ndc-duckduckapi/values.yaml

```diff
@@ -0,0 +1,123 @@
+useReleaseName: true
+
+additionalAnnotations: |
+  checksum/config: {{ include (print $.Template.BasePath "/secret.yaml") . | sha256sum }}
+  app.kubernetes.io/access-group: connector
+
+networkPolicy:
+  ingress:
+    enabled: true
+    allowedApps:
+      - v3-engine
+
+# Container Configs
+image:
+  repository: ""
+  tag: ""
+  pullPolicy: IfNotPresent
+replicas: "1"
+wsInactiveExpiryMins: "1"
+securityContext:
+  runAsNonRoot: true
+  runAsGroup: 1000
+  runAsUser: 1000
+  fsGroup: 1000
+
+initContainers:
+  gitSync:
+    enabled: false
+    repo: "git@github.com:<org>/<repo>"
+    branch: "main"
+    secretName: ""
+    depsPackaged: true
+
+healthChecks:
+  enabled: false
+  livenessProbePath: "/healthz"
+  readinessProbePath: "/healthz"
+
+command: |
+  - bash
+  - /scripts/run.sh
+
+extraVolumes: |
+  - name: run-script
+    configMap:
+      name: {{ printf "%s-config" (include "common.name" .) }}
+
+extraVolumeMounts: |
+  - name:  run-script
+    mountPath: /scripts/run.sh
+    subPath: run.sh
+
+hpa:
+  enabled: false
+  minReplicas: 2
+  maxReplicas: 4
+  metrics:
+    - type: Resource
+      resource:
+        name: cpu
+        target:
+          type: Utilization
+          averageUtilization: 50  # Target 50% CPU utilization per pod
+    - type: Resource
+      resource:
+        name: memory
+        target:
+          type: Utilization
+          averageUtilization: 80  # Target 80% memory utilization per pod
```

Comment:
Let's remove this everywhere? This horizontal scaling based on memory doesn't make sense